src/web/mjs/connectors/AtelierDuNoir.mjs

```diff
@@ -1,49 +0,0 @@
-import FoolSlide from './templates/FoolSlide.mjs';
-
-/**
- *
- */
-export default class AtelierDuNoir extends FoolSlide {
-
-    /**
-     *
-     */
-    constructor() {
-        super();
-        super.id = 'atelierdunoir';
-        super.label = 'AtelierDuNoir';
-        this.tags = [ 'manga', 'high-quality', 'english', 'scanlation' ];
-        this.url = 'http://atelierdunoir.org';
```

Comment:
New website at https://atelierdunoir.wordpress.com, but only contains a cross link to MangaDex and direct download links... so no need to keep this in HakuNeko